### DISCUSSION TITLE
Indonesian (Indonesia)

### DISCUSSION THREAD

**Discussion by abengkris** [sovenor/bitcoin-rocks#34] [Translations]
Can i add Indonesian language?

**Comment by sovenor** [comment]
Yes, that would be amazing! Please follow our Translations Guide here: https://github.com/sovenor/bitcoin-rocks/blob/main/CONTRIBUTING.md

**Comment by sovenor** [comment]
Just wanted to make a note here that Indonesian is available across the entire website, and has been for a while thanks to @abengkris!
